docs/user-guide/configure-overview.md

```diff
@@ -1,14 +1,17 @@
-## Configuring Styx environment
+## Configuring Styx
 
```

Comment:
Hi. I believe this file has incorrect information about load balancer configurations. In particular:

```
    loadBalancing:
      # Load balancer strategy type. Allowed values: "LEAST\_RESPONSE\_TIME", "ROUND\_ROBIN", and "ADAPTIVE". Defaults to "ROUND_ROBIN".
      strategy: "ADAPTIVE"
```

I believe you'd have to supply the full class name using the factory. See the load balancer config page for an example.